osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/services/DeviceMessageLoggingService.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.iec60870.services;
+
+import org.opensmartgridplatform.adapter.protocol.iec60870.domain.valueobjects.DeviceMessageLog;
+import org.opensmartgridplatform.adapter.protocol.iec60870.infra.messaging.LogItemRequestMessage;
+import org.opensmartgridplatform.adapter.protocol.iec60870.infra.messaging.LogItemRequestMessageSender;
+import org.opensmartgridplatform.shared.infra.jms.MessageMetadata;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DeviceMessageLoggingService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeviceMessageLoggingService.class);
+
+    private static LogItemRequestMessageSender iec60870LogItemRequestMessageSender;
+
+    @Autowired
+    public DeviceMessageLoggingService(final LogItemRequestMessageSender iec60870LogItemRequestMessageSender) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1118)